projects/WTI-API/src/main/config/ServerInit.java

```diff
@@ -154,18 +159,25 @@ public static void updateUIAppConfig() {
 	
 	/**
 	 * Returns a String containing the local IP address of the machine on which the WTI server is running.
-	 * The returned address is obtained by calling {@link InetAddress#getHostAddress()} on the address
-	 * returned by {@link DatagramSocket#getLocalAddress()}.
+	 * The returned address is initially obtained by calling {@link InetAddress#getHostAddress()} on the address
+	 * returned by {@link DatagramSocket#getLocalAddress()}. This is the address returned unless the pc2v9.ini
+	 * file contains an entry "wtiOverridePublicIP", in which case the value of that entry is returned instead.
 	 * 
-	 * @return a String containing an IP address
+	 * @return a String containing an IP address, or null if an exception occurs while fetching the local IP address.
 	 */
 	public static String getLocalIp() {
 		// Source: https://stackoverflow.com/questions/9481865/getting-the-ip-address-of-the-current-machine-using-java
 		try (final DatagramSocket socket = new DatagramSocket()) {
 			socket.connect(InetAddress.getByName("8.8.8.8"), 10002);
-			return socket.getLocalAddress().getHostAddress();
+			String myIP = socket.getLocalAddress().getHostAddress();
+			if (publicIPOverride!=null) {
+			    myIP = publicIPOverride;
+			}
+			return myIP ;
```

Comment:
why bother do the socket.connect if there is a publicOverride?

Reply:
I thought of that after the fact.  Since it only happens once I decided not to go back and fix it, favoring getting it to Ahmed timely.  I can change that if you think it's important...

Reply:
I just pushed an update fixing this.